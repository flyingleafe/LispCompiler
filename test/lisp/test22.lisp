(let ((x 5))
  (progn
    (printString "Current value: 5")
    (printString "Add some number:")
    (let ((y (readInt)))
      (progn
        (set x (+ x y))
        (printString "New value:")
        (printInt x)))))
