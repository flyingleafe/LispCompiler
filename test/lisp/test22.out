Current value: 5
Add some number:
New value:
11
